packages/field/src/useField.ts

```diff
@@ -8,7 +8,16 @@
 import { useState } from 'react';
 import { generateId } from '@zendeskgarden/container-utilities';
 
-export function useField(idPrefix) {
+export interface IUseFieldPropGetters {
+  getHintProps: <T>(options?: T & React.HTMLProps<any>) => React.HTMLProps<any>;
```

Comment:
@hzhu this interface may need some tweaking to allow for custom props being provided. Compare this to the `useSelection` hook I believe it should use

```tsx
getHintProps: <T>(options?: T) => T & React.HTMLProps<any>
```

Reply:
@austin94 thanks for the feedback! I've created a new PR to incorporate these suggestions, #112. Let me know how that looks.